apps/docs/stories/components/ModularCheckout/Docs.mdx

```diff
@@ -0,0 +1,189 @@
+
+**Full billing address:**
+
+```javascript
+{
+  name: string;
+  address_line1: string;
+  address_line2?: string;
+  address_city: string;
+  address_state: string;
+  address_postal_code: string;
+}
+```
+
+**Only postal code:**
+
+```javascript
+{
+  address_postal_code: string;
+}
+```
+
+### Card Form Billing
+
+When using `<justifi-card-form />`, billing information **must be provided** through either:
+
+- `<justifi-billing-information-form />`
+
+- `<justifi-postal-code-form />`
+
+- Passing the full `billingInformation` object directly to the `submitCheckout` method
+
+- Passing the `billingFormInformation` object with only `address_postal_code` to the `submitCheckout` method
+
+### Bank Account Form Billing:
+
+When using `<justifi-bank-account-form />`, billing information **must be provided** through either:
+
+- `<justifi-billing-information-form />`
+
+- Passing the full `billingInformation` object directly to the `submitCheckout` method
+
+## Completing Checkout
+
+To finalize the checkout process, include a submit button that calls the `submitCheckout` method on the wrapper. This method.
+
+- Handles validation, form submission and payment method tokenization.
+
+- Optionally accepts a `billingInformation` object (useful when not using a billing component)
+
+- Emits a `checkout-complete-event` when the submission is successful
+
+- This method **does not** return a promise
+
+> If you passed a `payment_method_group_id` to the create checkout API and you want to give the user the option to save a newly entered payment method to that payment method group for future use, set the `savePaymentMethod` prop to true on the `justifi-modular-checkout`. This will automatically save the payment method to that payment method group after successful checkout.
+
+## Validation
+
+When `submitCheckout` is called, the wrapper automatically validates all included subcomponents. If any required fields are missing or invalid, submission is blocked and validation errors will appear inline.
+
+You can also trigger validation manually by calling the `validate` method. This returns a promise that resolves to a `boolean` indicating whether the form is valid.
+
+# Props, events and methods
+
+---
+
+<ArgTypes exclude={['Theme', 'slot']} />
+
+# Authorization
+
+---
+
+<p>
+  Authorization is performed by passing a web component token as{' '}
+  <code>auth-token</code>.
+</p>
+<ul>
+  <li>
+    <strong>Web Component Token</strong>: These tokens are generated by your
+    backend services using the [Web Component Tokens
+    API](https://docs.justifi.tech/api-spec#tag/Web-Component-Tokens/operation/CreateWebComponentToken).
+    Each token can be scoped to perform a set number of actions and is active
+    for 60 minutes. When creating a web component token for this specific
+    component you'll need to use the role: `read:account:account_id` or
+    `write:account:account_id`. Make sure the value for `account_id` matches the
```

Comment:
Does this web component not need the same web component roles as the other checkout component? Those are:
`write:checkout:checkout_id` - use the checkout_id you receive when you create a checkout via [Checkout API](https://docs.justifi.tech/api-spec#tag/Checkouts)
`write:tokenize:account_id` - use the account_id you pass to the checkout API